pkg/locker/README.md

```diff
@@ -0,0 +1,65 @@
+Locker
```

Comment:
Probably doesn't make a difference here, but this was added in https://github.com/docker/docker/pull/17185, which is not in the 1.9.x release